README.md

```diff
@@ -1,51 +1,13 @@
-# MazeLike
+# ![Mazelike logo](Frontend/img/mazelike_logo.png)
+[![Build Status](https://travis-ci.com/MazelikeGame/mazelike.svg?branch=master)](https://travis-ci.com/MazelikeGame/mazelike)
 
-## Run the server
+# Multiplayer video game played in a browser.
 
-Basic demo
-```
-docker run -p 3000:3000 ryan3r/mazelike
-```
+Players can join a lobby of up to four players. Players are then tasked with
+fighting mobs and defeating bosses in order to progress through randomly
+generated rooms/levels.
 
-Persistant changes
-```
-docker volume create mazelike
-docker run -p 3000:3000 -v mazelike:/data ryan3r/mazelike
-```
+### Documentation
+  * [JSDocs](https://mazelike.ryan3r.com/docs/)

```

Comment:
This should probably be moved to [github pages](https://pages.github.com/).  We would also probably want a git hook or something to make sure the docs are updated before we push.